src/main/resources/templates/subscription.html

```diff
@@ -156,7 +156,22 @@ <h3 class="modal-title text-center" id="formHeader">Subscription Form</h3>
                                     </label>
                                 </div>
                             </div>
-
+                            
+                             <!-- Form post emailSubject field -->                            
+                            <div class="form-group" data-bind="visible: $data.notificationType() == 'MAIL'">
+                                <label class="pl-1 control-label font-weight-bold">Subject</label>
+                                <img class="cursor-pointer popup-large" id="restPostMediaTypeInfo" alt="RestPostMediaType Info" src="assets/images/information.png" data-toggle="tooltip" data-placement="top"
+                                    title=
+                                        "This is your choice for the email notification subject. <br>
+                                        Default value is 'Email Subscription Notification' <br>
```

Comment:
You should not say what a default value is, just point that a default value may have been set by the admin, If the default value has been changed by the admin we lie to the users here.

Reply:
I have changed the text ...